custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -369,6 +369,7 @@
             "label": "Lemmatize",
             "description": "Simplify words to their lemma form e.g. going → go, mice → mouse",
             "type": "BOOLEAN",
+            "visibilityCondition": "model.language == 'language_column' || ['es', 'nb', 'fr', 'de', 'ru', 'pl','ca','cs','da','hr','hu','id','it','lb','lt', 'pt', 'ro','sr','tl','tr', 'ur', 'en','mk','nl','sv','bn','el','fa'].includes(model.language)",
```

Comment:
According to the conversation with Adrien and Krishna, let's add 2 lemmatize button:
1. One which appears in monolingual mode only for languages that support lemmatization
2. One which appears in multilingual mode. This one will look the same as (1) but have an additional sentence in the description saying something like "X languages do not support lemmatization"

Reply:
Did it in commit 04fd486 ! Tell me WYT of the message :)

Reply:
Made a slight rewording in 074f0fa